### PR TITLE
Postgres feature

### PR DESCRIPTION
**Pull request by alextsaihi**
migrated from [Installation of PostgreSQL via Dockerfile #80](https://github.com/ga4gh/ga4gh-starter-kit-drs/pull/80)

### CODE REVIEW

**Comment by alextsaihi** [comment]
hi @dashrath-chauhan and @yash-puligundla , I've migrated the PR over to this one so that it is a branch of ga4gh/ga4gh-starter-kit-drs instead of a fork

old PR: #80

**Comment by alextsaihi** [comment]
steps to test:

FROM POSTGRES

docker-compose up -d
from postgres container's terminal, psql -U postgres
\c starter_kit_db -> you should see "You are now connected to database `starter_kit_db` as user `postgres`"
\dt-> you should see 9 tables listed
\q (to exit postgres terminal)

FROM DRS
from Postman, GET the following urls:
`http://localhost:4500/ga4gh/drs/v1/service-info`
`http://localhost:4500/ga4gh/drs/v1/objects/1a570e4e-2489-4218-9333-f65549495872`

CLOSING TEST
stop postgres container and try a GET request again

File: README.md [hunk 1]
```diff
@@ -35,6 +37,38 @@ OR, run container with config file overriding defaults
...
+
+`docker compose -f docker-compose-postgres.yml up -d`
+
+To ensure that both Starter-Kit DRS and Postgres containers are running:
+
+`docker ps`
+
+This command should list both containers with names `drs` and `postgres`
+
+From the DRS config file > databaseProps attribute, we map DRS to the PostgreSQL databse using its jdbc URL as well as Postgres credentials (username/password)
```
Comment (yash-puligundla) [inline README.md]:
```suggestion
From the DRS config file > databaseProps attribute, we map DRS to the PostgreSQL database using its jdbc URL as well as Postgres credentials (username/password)
```

**Comment by yash-puligundla** [comment]
Good work Alex! Everything works as expected.

**Pull request merged**